### PR TITLE
Special US election thank you page

### PR DESCRIPTION
**Pull request by TomPretty**
## Why are you doing this?

Implement the special US election thank you page newsletter sign up as an ab test. As per [this card](https://trello.com/c/e65OkQs6/2365-us-election-special-thank-you-page).

## Screenshots
desktop:
Screenshot 2020-10-27 at 11 37 24

mobile: 
Screenshot 2020-10-27 at 11 38 31

### CODE REVIEW

File: support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouUsElectionNewsletter.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+const signUpToNewsLetter = (email: string) => {
+  const formData = new FormData();
+  formData.append('email', email);
+  formData.append('listName', 'us-morning-newsletter');
+  formData.append('name', '');
+
+  fetch('https://www.theguardian.com/email', {
+    method: 'POST',
+    body: new URLSearchParams(formData),
```
Comment (tomrf1) [inline support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouUsElectionNewsletter.jsx:27]: is the `URLSearchParams` necessary here?

File: support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouUsElectionNewsletter.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  });
+};
+
+type ContributionThankYouUsElectionNewsletterProps = {|
+  email: string
+|};
+
+const ContributionThankYouUsElectionNewsletternsent = ({
```
Comment (tomrf1) [inline support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouUsElectionNewsletter.jsx:35]: `ContributionThankYouUsElectionNewsletter`?

Comment (TomPretty) [reply support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouUsElectionNewsletter.jsx]: yep!

File: support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouUsElectionNewsletter.jsx (hunk 1 above)
Comment (TomPretty) [reply support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouUsElectionNewsletter.jsx:27]: It seems to be, I'm not 100% sure why that's the case though. It was surprisingly difficult to find good examples of using fetch to post form data - it's just all json I guess

Comment (TomPretty) [reply support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouUsElectionNewsletter.jsx:27]: I got it from [here](https://stackoverflow.com/a/46640266/11995118)

**Review by tomrf1 — APPROVED** [review]
great work!

**Pull request merged**